ci/test.sh

```diff
@@ -2,5 +2,5 @@
 
 set -ex
 
-pip install tox
+pip install tox colorama==0.4.4
```

Comment:
No objection per se, why do we have pin for a specific version? My fear is forgetting to update this version and it lagging behind? Is it possible to at least put these dependencies in a `ci-reqs.txt` file for Python so that Dependabot could be aware of it? I'm not even sure if it works that way

Reply:
tox requires a specific version of awscli, which in turn requires a specific version of colorama. Python unfortunately can't resolve the dependency on its own. We do have requirements.txt files, but tox is only needed when running the test scripts, so I think that's why it was originally separated out.

Reply:
sounds good, I've updated the code for this